modules/reference/appvms/appflowy.nix

```diff
@@ -12,6 +17,15 @@
     {
       hardware.graphics.enable = true;
       time.timeZone = config.time.timeZone;
+      ghaf.givc.appvm =
+        let
+          prefix = "/run/current-system/sw/bin";
```

Comment:
should this prefix + run-waypipe be some common configuration, so just `config.runIt` or something similar?

Reply:
Could be indeed. Can set this as a flag in common, and use it in the VMs.